mvc-api/mvc-service/conf/routes

```diff
@@ -6,6 +6,8 @@ GET		/health						controllers.HealthController.health()
 #POST    /v2/search						controllers.SearchController.search()
 #POST    /v3/search						controllers.SearchController.search()
 POST    /v3/mvcsearch                   controllers.SearchController.search()
+POST    /v3/mvccontent/create/json      controllers.MVCContentController.createUsingJson()
+POST    /v3/mvccontent/create/csv       controllers.MVCContentController.createUsingCSV()
```

Comment:
Change the Path to `/v3/mvccontent/import`

Reply:
okay